index.html

```diff
@@ -8,28 +8,43 @@
     <script src="https://kit.fontawesome.com/d628e85c59.js" crossorigin="anonymous"></script>
     <title>Andisiwe Portfolio</title>
 </head>
-<body class="portfolio">
-                                        <!--Task-bar-->
+<body class="portfolio">                                        <!--Task-bar-->                        
     <header>
         <nav class="nav">
             <div class="container">
                 <div> 
-                    <a class="Mylogo" href="#"> Andisiwe</a>
+                    <a class="Mylogo" href="#">Andisiwe</a>
                 </div>
-                <div>
-                    <a href="">
-                        <i class="fa-solid fa-bars"></i>
+                                <div>
+                                    <a href="">
+                                        <i class="fa-solid fa-bars"></i>
+                                    </a>
+                                </div>
+            <ul class="nav-links">
+                <li><a href="#" class="portfolio">Portfolio
+                    </a>
+                </li>
+                <li><a href="#" class="portfolio">About
+                </a>
+                </li>
+                <li><a href="#" class="portfolio">Contact
+                </a>
+                </li>
+                <li><a href="">
+                        <img src="Images/mail.svg" class="mail">
                     </a>
+                </li>      
+            </ul>
                 </div>
-            </div>
         </nav>
     </header>
-                                                    <!--Heading-intro-->
+                                                    <!--Heading-intro-->                   
 <section class="introduction">
+    <img src="Images/big-img.svg">
```

Comment:
As described in the past week, the CSS `background-image` property must be used for the illustrations. So please make sure to update this.